wde.go

```diff
@@ -55,8 +55,17 @@ recommended that your main function look like the the code below.
 
 wde.Run() will return when wde.Stop() is called.
 
-For this to work, you must import one of the go.wde backends. For
-instance,
+For this to work, you must import one of the go.wde backends. To
+automatically import the correct backend, add this to your imports:
+
+        import _ "github.com/skelterjohn/go.wde/init"
+
+This will register a backend with go.wde, allowing you to call
+wde.Run(), wde.Stop() and wde.NewWindow() without referring to the
+backend explicitly.
+
+If you would like to disable a backend, you must import all
```

Comment:
I don't understand this part at all. You should only ever import one backend. It so happens that the init package uses build rules to import one based on the platform.